dev/set_matrix.py

```diff
@@ -434,37 +434,38 @@ def expand_config(config):
         flavor = flavor_key.split("-")[0]
         package_info = cfgs.pop("package_info")
         all_versions = get_released_versions(package_info["pip_release"])
+        pip_release = package_info["pip_release"]
 
         for key, cfg in cfgs.items():
-            # Released versions
-            min_ver = cfg["minimum"]
-            max_ver = cfg["maximum"]
-            versions = filter_versions(all_versions, min_ver, max_ver, cfg.get("unsupported"),)
-            versions = select_latest_micro_versions(versions)
-
-            # Explicitly include the minimum supported version
-            if min_ver not in versions:
-                versions.append(min_ver)
-
-            pip_release = package_info["pip_release"]
-            for ver in versions:
-                job_name = " / ".join([flavor_key, ver, key])
-                requirements = ["{}=={}".format(pip_release, ver)]
-                requirements.extend(process_requirements(cfg.get("requirements"), ver))
-                install = make_pip_install_command(requirements)
-                run = remove_comments(cfg["run"])
-
-                matrix.append(
-                    Hashabledict(
-                        flavor=flavor,
-                        job_name=job_name,
-                        install=install,
-                        run=run,
-                        package=pip_release,
-                        version=ver,
-                        supported=Version(ver) <= Version(max_ver),
+            if "minimum" in cfg and "maximum" in cfg:
```

Comment:
Added this line to skip searching for versions when `cfg` is missing `minimum` and `maximum` fields.